src/coreclr/nativeaot/nativeaot.sln

```diff
@@ -260,4 +184,7 @@ Global
 	GlobalSection(ExtensibilityGlobals) = postSolution
 		SolutionGuid = {81B16D59-928B-49C1-839D-10E4747B0DC0}
 	EndGlobalSection
+	GlobalSection(SharedMSBuildProjectFiles) = preSolution
+		..\..\libraries\System.Private.CoreLib\src\System.Private.CoreLib.Shared.projitems*{e4bc768b-f97d-4a8f-9391-b65df3eb47c6}*SharedItemsImports = 5
+	EndGlobalSection
```

Comment:
`.sln` change not reverted intentionally?

Reply:
Reverted